packages/react-native-sdk/src/components/Call/CallTopView/ParticipantsInfoBadge.tsx

```diff
@@ -90,6 +91,7 @@ export const ParticipantsInfoBadge = ({
             typefaces.subtitle,
             participantInfoBadge.participantsCountText,
           ]}
+          testID={ButtonTestIds.PARTICIPANTS_COUNT_TEXT}
```

Comment:
For some reason, iOS does not see the text inside this element, but it sees it in the parent element. That's why I had to add two different locators and access them in the tests through regex `participants-count-.*`.